### ISSUE TITLE
Cannot uninstall the kernel plugin

### ISSUE THREAD

**Issue by Mister-XY** [OpenMediaVault-Plugin-Developers/openmediavault-kernel#4]
Hello, I have installed the OMV kernel plugin and would like to uninstall it again. Unfortunately it is not possible because the uninstallation is aborted due to too many errors.

`Reading package lists...
Building dependency tree...
Reading state information...
The following packages will be REMOVED:
  openmediavault-kernel
0 upgraded, 0 newly installed, 1 to remove and 0 not upgraded.
1 not fully installed or removed.
After this operation, 64.5 kB disk space will be freed.
(Reading database ... 
(Reading database ... 5%
(Reading database ... 10%
(Reading database ... 15%
(Reading database ... 20%
(Reading database ... 25%
(Reading database ... 30%
(Reading database ... 35%
(Reading database ... 40%
(Reading database ... 45%
(Reading database ... 50%
(Reading database ... 55%
(Reading database ... 60%
(Reading database ... 65%
(Reading database ... 70%
(Reading database ... 75%
(Reading database ... 80%
(Reading database ... 85%
(Reading database ... 90%
(Reading database ... 95%
(Reading database ... 100%
(Reading database ... 93200 files and directories currently installed.)
Removing openmediavault-kernel (6.4.6) ...
/var/lib/dpkg/info/openmediavault-kernel.postrm: 34: remote_action: not found
dpkg: error processing package openmediavault-kernel (--remove):
 installed openmediavault-kernel package post-removal script subprocess returned error exit status 127
dpkg: too many errors, stopping
Errors were encountered while processing:
 openmediavault-kernel
Processing was halted because there were too many errors.

** CONNECTION LOST **`

And that comes directly from OMV

```Failed to execute command 'export PATH=/bin:/sbin:/usr/bin:/usr/sbin:/usr/local/bin:/usr/local/sbin; export LANG=C.UTF-8; export LANGUAGE=; export DEBIAN_FRONTEND=noninteractive; apt-get --yes --auto-remove --purge remove openmediavault-kernel 2>&1' with exit code '100': Reading package lists...

Building dependency tree...

Reading state information...

The following packages will be REMOVED:
  openmediavault-kernel
0 upgraded, 0 newly installed, 1 to remove and 0 not upgraded.
1 not fully installed or removed.
After this operation, 64.5 kB disk space will be freed.
(Reading database ...
(Reading database ... 5%
(Reading database ... 10%
(Reading database ... 15%
(Reading database ... 20%
(Reading database ... 25%
(Reading database ... 30%
(Reading database ... 35%
(Reading database ... 40%
(Reading database ... 45%
(Reading database ... 50%
(Reading database ... 55%
(Reading database ... 60%
(Reading database ... 65%
(Reading database ... 70%
(Reading database ... 75%
(Reading database ... 80%
(Reading database ... 85%
(Reading database ... 90%
(Reading database ... 95%
(Reading database ... 100%
(Reading database ... 93200 files and directories currently installed.)
Removing openmediavault-kernel (6.4.6) ...
/var/lib/dpkg/info/openmediavault-kernel.postrm: 34: remote_action: not found
dpkg: error processing package openmediavault-kernel (--remove):
 installed openmediavault-kernel package post-removal script subprocess returned error exit status 127
dpkg: too many errors, stopping
Errors were encountered while processing:
 openmediavault-kernel
Processing was halted because there were too many errors.
E: Sub-process /usr/bin/dpkg returned an error code (1)

OMV\ExecException: Failed to execute command 'export PATH=/bin:/sbin:/usr/bin:/usr/sbin:/usr/local/bin:/usr/local/sbin; export LANG=C.UTF-8; export LANGUAGE=; export DEBIAN_FRONTEND=noninteractive; apt-get --yes --auto-remove --purge remove openmediavault-kernel 2>&1' with exit code '100': Reading package lists...

Building dependency tree...

Reading state information...

The following packages will be REMOVED:
  openmediavault-kernel
0 upgraded, 0 newly installed, 1 to remove and 0 not upgraded.
1 not fully installed or removed.
After this operation, 64.5 kB disk space will be freed.
(Reading database ...
(Reading database ... 5%
(Reading database ... 10%
(Reading database ... 15%
(Reading database ... 20%
(Reading database ... 25%
(Reading database ... 30%
(Reading database ... 35%
(Reading database ... 40%
(Reading database ... 45%
(Reading database ... 50%
(Reading database ... 55%
(Reading database ... 60%
(Reading database ... 65%
(Reading database ... 70%
(Reading database ... 75%
(Reading database ... 80%
(Reading database ... 85%
(Reading database ... 90%
(Reading database ... 95%
(Reading database ... 100%
(Reading database ... 93200 files and directories currently installed.)
Removing openmediavault-kernel (6.4.6) ...
/var/lib/dpkg/info/openmediavault-kernel.postrm: 34: remote_action: not found
dpkg: error processing package openmediavault-kernel (--remove):
 installed openmediavault-kernel package post-removal script subprocess returned error exit status 127
dpkg: too many errors, stopping
Errors were encountered while processing:
 openmediavault-kernel
Processing was halted because there were too many errors.
E: Sub-process /usr/bin/dpkg returned an error code (1) in /usr/share/openmediavault/engined/rpc/pluginmgmt.inc:156
Stack trace:
#0 /usr/share/php/openmediavault/rpc/serviceabstract.inc(619): Engined\Rpc\PluginMgmt->Engined\Rpc\{closure}('/tmp/bgstatusHl...', '/tmp/bgoutputho...')
#1 /usr/share/openmediavault/engined/rpc/pluginmgmt.inc(160): OMV\Rpc\ServiceAbstract->execBgProc(Object(Closure))
#2 [internal function]: Engined\Rpc\PluginMgmt->remove(Array, Array)
#3 /usr/share/php/openmediavault/rpc/serviceabstract.inc(123): call_user_func_array(Array, Array)
#4 /usr/share/php/openmediavault/rpc/rpc.inc(86): OMV\Rpc\ServiceAbstract->callMethod('remove', Array, Array)
#5 /usr/sbin/omv-engined(537): OMV\Rpc\Rpc::call('Plugin', 'remove', Array, Array, 1)
#6 {main} ```

**Comment by ryecoaaron** [comment]
I fixed this the other day but haven’t pushed a new version yet. I’m on vacation and can’t push the new package. Just reinstall the plugin for another week. I will hurt nothing and use no resources.

**Comment by ryecoaaron** [comment]
Fixed in 6.4.7 in repo now.
